ckb-bench/src/bench.rs

```diff
@@ -60,6 +84,18 @@ impl LiveCellProducer {
                     self.seen_out_points
                         .put(cell.out_point.clone(), Instant::now());
                     let _ignore = live_cell_sender.send(cell);
+                    count += 1;
+                    duration_count += 1;
+                    if Instant::now().duration_since(start_time) >= Duration::from_secs(log_time) {
+                        ckb_testkit::info!("[LiveCellProducer] producer count: {} ,duration time:{}s, duration tps:{} ", count,log_time,duration_count/log_time);
+                        duration_count = 0;
+                        start_time = Instant::now();
+                    }
+                }
+
+                if fist_send_finished {
+                    fist_send_finished = true;
+                    self.seen_out_points.resize(count + 10)
```

Comment:
We defined `fist_send_finished = false` on above. But we never set it to `true`, so this condition will never reach.